src/main/java/seedu/duke/command/ListIngredientCommand.java

```diff
@@ -9,6 +9,6 @@
 public class ListIngredientCommand extends Command{
     @Override
     public void execute(Menu menu, Ui ui) {
-
+        ui.printIngredients(menu, index);
     };
```

Comment:
remove redundant semicolon

```suggestion
    }
```